src/NuGet.Indexing/Extraction/CatalogPackageReader.cs

```diff
@@ -46,6 +46,24 @@ public override IEnumerable<string> GetFiles()
             }
         }
 
+        public override IReadOnlyList<PackageType> GetPackageTypes()
```

Comment:
There are tests in the NuGet.IndexingTests/Extraction/CatalogPackageMetadataExtractorTests.cs that test this code path.

---
In reply to: [78818424](https://github.com/NuGet/NuGet.Services.Metadata/pull/87#discussion_r78818424) [](ancestors = 78818424)